### ISSUE TITLE
OpenRA doesn't start, but computer restarts

### ISSUE THREAD

**Issue by timpeteler** [OpenRA/OpenRA#11266]
I installed the latest OpenRA build (release-20160508) on my Windows 8.1 64 bit computer.
When I double click the OpenRA icon on my desktop, the screen turns white and 3 seconds later, my computer restarts.

The log files in my user home directory are all empty, except perf.log.
[OpenRA.zip](https://github.com/OpenRA/OpenRA/files/262294/OpenRA.zip)

**Comment by abcdefg30** [comment]
`perf.log` only contains spaces as it seems.
Just some standard questions: Did you check that your antivirus doesn't block the game (although that shouldn't make the computer restart)? And did you already try redownloading and reinstalling the game? As you mention it explicitly, does something else happen when you don't start OpenRA from your desktop (but directly from the main game directory)?

**Comment by GraionDilach** [comment]
Is OpenRA the sole performance-heavy application you run on that computer? Honestly, I could also think of the overheat-protection restarting the PC.

**Comment by timpeteler** [comment]
Some more detailed information about my computer: Dell E7440, Intel i7, 16GB RAM, SSD
Temperature is not the issue here I guess.

I don't have any antivirus software running except Windows Defender. But the restart issue is still there, even if I disable Defender.

I also downloaded and installed OpenRA twice.

The issue is the same, even if I directly start OpenRa.exe from the installation directory and not from the desktop link.

**Comment by pchote** [comment]
OpenRA itself doesn't have the ability to force a restart, so this is probably caused by a bug in your GPU driver corrupting the kernel memory.  What GPU and driver are you using?  You should make sure your drivers are up to date (the ones included with Windows aren't good enough – you need to download them from the hardware vendor).

**Comment by timpeteler** [comment]
I have a Intel HD Grafic Card... so basically no "real" GPU.
I downloaded the latest drivers directly from Intel, but that didn't help.
I now upgraded to Windows 10 + got the latest drivers and now the game starts.
Even though in fullscreen mode, I can't do anything, because the game is just "frozen". 
I can't click anything... by switching between the desktop and the game back and forth I managed to configure windowed mode... and this seems to be running fine now.

Maybe one more thing that might have had an impact: I'm running a U-HD monitor with 3840px x 2160px.
